api/services/course.js

```diff
@@ -86,8 +88,62 @@ const getCoursesForCarousel = async () => {
   });
 };
 
+const postSignatureUserCourse = async (courseId, authorization) => {
+  if (
```

Comment:
Please, just write a check on the course existing.

Reply:
Done